mne/time_frequency/tfr.py

```diff
@@ -333,10 +351,11 @@ def _time_frequency(X, Ws, use_fft):
         tfrs = _cwt_convolve(X, Ws, mode)
 
     for tfr in tfrs:
-        tfr_abs = np.abs(tfr)
-        psd += tfr_abs ** 2
```

Comment:
After a period of introspection (and discussion with @dengemann) I realized the original is correct. It looks like the data shouldn't necessarily be the same, despite what we talked about before. Reverting a few changes...